docs/popular-topics/error-handling.mdx

```diff
@@ -0,0 +1,356 @@
+---
+title: Error Handling
+description: All about handling errors.
+---
+
+import {
+    DiscordEmbed,
+    DiscordInteraction,
+    DiscordMessage,
+} from "discord-message-components/packages/react";
+import DiscordComponent, { defaultOptions } from "../../src/components/DiscordComponent";
+
+## About
+
+When using the library, errors are frequently raised due to various reasons. These reasons range from the Discord API 
+failing your bot's request due to missing permissions or an invalid syntax error.
+
+Before going into error handling, it's best that you learn the basic ways of handling errors. A good resource is:
+
+-   [W3School's Guide to Python Try-Except Statements](https://www.w3schools.com/python/python_try_except.asp)
+
+:::note
+As of writing this guide page, bridge commands do not currently support per-command error handling. 
+
+For these types of commands, it is recommended to utilize global error handling 
+until [the feature](https://github.com/Pycord-Development/pycord/issues/1388) is added.
+:::
+
+## Basic Handling
+
+The most basic way to handle errors is to tackle it at the source. This can be done like so:
+
+:::tip Multiple Errors
```

Comment:
This tip feels a little misplaced

Reply:
Felt so too, will probably move that as well as the mixing handlers tip into a small FAQ section.